xtask/src/commands/compliance.rs

```diff
@@ -63,7 +75,7 @@ impl Compliance {
 
         let licences_html_after = Self::digest_for_license_file()?;
 
-        (licenses_html_before == licences_html_after).then(|| {
+        (licenses_html_before != licences_html_after).then(|| {
```

Comment:
This seems unexpected.

Reply:
This is actually a fix, the initial intention was to only display the message "hey your license file is now up to date, please check it in" if there were actual changes to be checked in 😓

Reply:
Yeah, okay, that makes sense.